### ISSUE TITLE
Camel-case tag helper directives

### ISSUE THREAD

**Issue by glen-84** [aspnet/Razor#264]
When multiple words are joined and the full string is lower-cased (or upper-cased), it makes it difficult to read.

For example:

```c#
@addtaghelper X
```

Would it be possible to make directives case insensitive, or at least allow camel or pascal case?

```c#
@addTagHelper X
```

Unrelated to this ticket: Would it not make sense to use a different prefix for directives, so that they don't look like variables? Something like:

```c#
@!model WebApplication1.Models.LoginViewModel
```

... just a thought.

**Comment by danroth27** [comment]
We think directives should be case sensitive, but we think we should change addtaghelper to addTagHelper, removetaghelper to removeTagHelper, etc.

**Comment by glen-84** [comment]
Sounds good.

**Comment by dougbu** [comment]
Changed title from "RFC: Make directives case insensitive or allow camel case"

**Comment by dougbu** [comment]
fixed in commits fbe331c1abcf840acab6c7f95e898f3e0ec37f66 and https://github.com/aspnet/Mvc/commit/451db6fb1621c0b67f1948085dd0bafe8092badf
